hibernate-core/src/main/java/org/hibernate/persister/entity/AbstractEntityPersister.java

```diff
@@ -166,6 +166,7 @@
 	private static final CoreMessageLogger LOG = CoreLogging.messageLogger( AbstractEntityPersister.class );
 
 	public static final String ENTITY_CLASS = "class";
+	public static final String VERSION_COLUMN_ALIAS = "v";
```

Comment:
To make this still unique, but more understandable/readable should we make this alias more like `version_`?  I'd suggest the same for `class` but I'm pretty sure places throughout the codebase refer to this name directly.

Reply:
`ENTITY_CLASS` has been set to `"class"` since 2007, and the contant is used in a bunch of places.